### PR TITLE
enhancement - display second sidenav panel next to first one

### PR DESCRIPTION
**Pull request by krlakwa**

### CODE REVIEW

File: src/alto-ui/SideNav/SideNav.scss [hunk 1]
```diff
@@ -348,7 +348,8 @@ $nav-btn-hover-text-dark: $white;
 .sidenav__panel {
   position: absolute;
   background: $white;
-  box-shadow: 1px 0 0 $coolgrey-20;
+  // border-right: 1px solid $coolgrey-20;
```
Comment (kevinmlsilva) [inline src/alto-ui/SideNav/SideNav.scss]: why the comment?

Comment (krlakwa) [reply src/alto-ui/SideNav/SideNav.scss]: I forgot to remove it. It's done now. Thanks

File: src/alto-ui/SideNav/SideNav.scss [hunk 2]
```diff
@@ -348,7 +348,7 @@ $nav-btn-hover-text-dark: $white;
 .sidenav__panel {
   position: absolute;
   background: $white;
-  box-shadow: 1px 0 0 $coolgrey-20;
+  box-shadow: inset -1px 0 0 $coolgrey-20;
```
Comment (djdeimon) [inline src/alto-ui/SideNav/SideNav.scss]: Let's find different way to hide this "border"

Comment (krlakwa) [reply src/alto-ui/SideNav/SideNav.scss]: I changed it

**Pull request merged**